rskj-core/src/main/java/co/rsk/net/eth/RskWireProtocol.java

```diff
@@ -103,56 +104,69 @@ public RskWireProtocol(RskSystemProperties config,
 
     @Override
     public void channelRead0(final ChannelHandlerContext ctx, EthMessage msg) throws InterruptedException {
-        loggerNet.debug("Read message: {}", msg);
+        String msgTraceId = TraceUtils.getRandomId();
+        String sessionId = Optional.ofNullable(this.channel)
+                .map(Channel::getPeerId)
+                .map(id -> StringUtils.substring(id,0, 14))
```

Comment:
can we wrap this into TraceUtils? Eg `co.rsk.util.TraceUtils#toId(String peerId)`, so that the id length would be incapsulated in one single class